sdk/core/azure-core-http-httpurlconnection/src/main/java/com/azure/core/http/httpurlconnection/HttpUrlConnectionClientBuilder.java

```diff
@@ -0,0 +1,19 @@
+package com.azure.core.http.httpurlconnection;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.util.logging.ClientLogger;
+
+public class HttpUrlConnectionClientBuilder {
```

Comment:
Refer to the other HTTP client implementations for some ideas on what should be set in here, and add to your backlog.